build.sbt

```diff
@@ -105,6 +106,12 @@ val doobie = module("doobie")
   )
   .dependsOn(core % "compile->compile;test->test")
 
+val cassandra = module("cassandra")
+  .settings(
+    libraryDependencies ++= Seq("com.datastax.oss" % "java-driver-core" % cassandraVersion)
```

Comment:
Any idea if we can test this module with an in-memory instance?

Reply:
I could try using testContainers. Never used before tho.

Reply:
Fine, we can try that later on

Reply:
Implemented testcontainers.